boost/plan.sh

```diff
@@ -25,15 +27,21 @@ pkg_build_deps=(
   core/libxslt
   core/openssl
   core/which
+  core/zlib
 )
 
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
 
 do_build() {
-  ./bootstrap.sh --prefix=$pkg_prefix
+  ./bootstrap.sh --prefix="$pkg_prefix"
 }
 
 do_install() {
-  ./b2 install --prefix=$pkg_prefix -q --debug-configuration -s NO_BZIP2=1
+  export NO_BZIP2=1
+  export ZLIB_LIBPATH
+  ZLIB_LIBPATH="$(pkg_path_for core/zlib)/lib"
+  export ZLIB_INCLUDE
+  ZLIB_INCLUDE="$(pkg_path_for core/zlib)/include"
```

Comment:
@robbkidd if you have a list of those core plans would you file an issue so we can get them fixed up? If not that's okay too.